manifests/init.pp

```diff
@@ -176,6 +176,9 @@
 #
 # $manage_squid::               boolean controls whether squid configuration is managed or manual
 #
+# $lazy_enable::                This controls whether redirect is enabled
```

Comment:
To me it's a bit confusing that first it's lazy, then it's redirect. You also have ```$lazy_enable``` where the code has ```$lazy_enabled``` (note the missing d)